### PR TITLE
ON-7125 # Added types for OneBlinkAppsError

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## oneblink__apps ([<kbd>unpkg</kbd>](https://unpkg.com/browse/oneblink__apps@latest/))
was missing the following properties:
1. useTenantCivicPlus
1. useTenantOneBlink
1. draftService
1. prefillService
1. paymentService

<details>
<summary>as well as these 1 other properties...</summary>
<p>jobService</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6c95c838904a0aee43ce64adce91c323557241e4
</p>

File: types/oneblink__apps/index.d.ts [hunk 1]
```diff
@@ -43,4 +43,25 @@ declare namespace authService {
...
+    title: string;
+    isOffline: boolean;
+    requiresAccessRequest: boolean;
+    requiresLogin: boolean;
+    httpStatusCode: number | undefined;
+    originalError: Error | undefined;
+
+    constructor(
+        message: string,
+        options: {
```
Comment (mymattcarroll) [inline types/oneblink__apps/index.d.ts:56]: This is also optional.

File: types/oneblink__apps/oneblink__apps-tests.ts [hunk 2]
```diff
@@ -44,3 +44,38 @@ const testAuthService = async () => {
     }
     await authService.logout();
 };
+
+// ONEBLINK APPS ERROR
+const testOneBlinkAppsError = () => {
+    // With no options
+    const errWithNoOptions = new OneBlinkAppsError('My New OneBlink Error', {});
```
Comment (mymattcarroll) [inline types/oneblink__apps/oneblink__apps-tests.ts:51]:
```suggestion
    const errWithNoOptions = new OneBlinkAppsError('My New OneBlink Error');
```

File: types/oneblink__apps/oneblink__apps-tests.ts [hunk 3]
```diff
@@ -44,3 +44,38 @@ const testAuthService = async () => {
...
+
+    // With no options
```
Comment (mymattcarroll) [inline types/oneblink__apps/oneblink__apps-tests.ts:53]:
```suggestion
    // With options
```

**Pull request merged**